Add limitation about CSS vars in SDK theming (#59238)
metabase/metabase: docs/embedding/sdk/appearance.md

## Limitations

> * CSS variables are not supported. Please upvote this [feature request](https://github.com/metabase/metabase/issues/59237), if you'd like support for this.
> * Colors configured in a question's visualization settings will override theme colors.

Reviewer: ```suggestion
- CSS variables aren't yet supported. If you'd like Metabase to support CSS variables, please upvote this [feature request](https://github.com/metabase/metabase/issues/59237).
- Colors set in the visualization settings for a question will override theme colors.
```
